gateway/1.8/modules/ROOT/pages/_partials/task-config-readiness-liveness.adoc

```diff
@@ -0,0 +1,191 @@
+
+To test whether a Flex Gateway instance is ready for incoming traffic, you can use the following CLI command:
+
+[source,ssh]
+----
+flexctl probe --check=readiness
+----
+
+If the Flex Gateway instance is ready to handle incoming traffic, the command returns an exit code of `0`. Otherwise the command returns an exit code of `1`.
+
+When Flex Gateway is started, its readiness state is `false`. The state changes to `true` after the following conditions are met:
+
+* API instances are deployed (at least one API must be configured)
+* Policies are initialized
+
+The readiness state is not impacted when you apply configuration changes to a running gateway instance. This ensures that the instance continues to receive traffic during configuration updates.
+
+Refer to the following for information about configuring readiness probes in specific environments:
+
+[tabs]
+====
+Kubernetes and OpenShift::
++
+By default, Flex Gateway running on Kubernetes includes a readiness probe configured in the Helm chart. The readiness probe automatically runs `flexctl probe --check=readiness --allow-api-errors --allow-envoy-errors --allow-policy-errors` at an interval of 10 seconds.
++
+The readiness probe is configured by default with the following values:
++
+[source,helm]
+----
+readinessProbe:
+ exec:
+   command:
+   - flexctl
+   - probe
+   - --check=readiness
+   - --allow-api-errors
+   - --allow-envoy-errors
+   - --allow-policy-errors
+ initialDelaySeconds: 10
+ periodSeconds: 10
+ failureThreshold: 2
+ timeoutSeconds: 5
+----
++
+[cols="1,2"]
+|===
+| Parameter | Description
+| `exec.command` | The readiness probe command
+| `--allow-api-errors`| The flag to allow errors in API instances
+| `--allow-envoy-errors`| The flag to allow Envoy configuration errors
+| `--allow-policy-errors`| The flag to allow policy configuration errors
+| `initialDelaySeconds` | The time in seconds to wait after startup before executing the first readiness probe
+| `periodSeconds` | The period in seconds between each readiness probe
+| `failureThreshold` | The number of failed readiness probes before the Kubernetes readiness status is set to `false`
+| `timeoutSeconds` | The time in seconds before the readiness probe times out
+|===
++
+To modify the default parameters, you must update your Helm chart. You can modify these parameters during or after the initial installation of the Helm chart. For more information about how to update a Helm chart, see xref:flex-gateway-k8-change-helm-settings.adoc[].
++
+NOTE: When using a readiness probe in Kubernetes (or Openshift) and Connected Mode, omit the `--wait` flag in the Helm chart used to deploy Flex Gateway. Not doing so might result in an installation failure because Helm waits for a readiness state of `true`. If there is no API deployed to your Flex Gateway instance, the readiness state is `false`. This is not an issue with custom readiness probe configurations or gateways that have an API configured.
```

Comment:
```suggestion
NOTE: When using a readiness probe in Kubernetes (or Openshift) and Connected Mode, omit the `--wait` flag in the Helm chart used to deploy Flex Gateway. Not doing so might result in an installation failure because Helm waits for a readiness state of `true`. If there's no API deployed to your Flex Gateway instance, the readiness state is `false`. This is not an issue with custom readiness probe configurations or gateways that have an API configured.
```